pom.xml

```diff
@@ -463,6 +455,12 @@
 			<artifactId>jackson-datatype-jsr310</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.commonmark/commonmark -->
+		<dependency>
+			<groupId>org.commonmark</groupId>
```

Comment:
Looked for a java markdown processor that will catch most of the standard syntax, and I'll remove any Synapse wiki widgets (which would look really strange in the search index).
https://mvnrepository.com/open-source/markdown-processors
Pegdown is the most popular, but it's at end-of-life (so no security fixes moving forward).  https://github.com/sirthias/pegdown